### ISSUE TITLE
Tooltip is not read by AVT tool JAWS

### ISSUE THREAD

**Issue by kavyanekkalapu** [carbon-design-system/carbon#1071]
## Detailed description

>We have an AVT issue in our project reported by JAWS in tooltips.
```On pressing insert+down arrow/down arrow, the tooltip information is not read by JAWS. Make sure all the information on the page is read by JAWS.```
we are using CC tooltip, i noticed that same issue exists in http://www.carbondesignsystem.com/components/tooltip/code.
> Is this issue related to a specific component?

Tooltip

> What did you expect to happen? What happened instead? What would you like to see changed?

On pressing insert+down arrow/down arrow, the tooltip information should be  read by JAWS. 

> What browser are you working in?

Firefox, chrome

> What version of the Carbon Design System are you using?

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
Cloud Automation Manager

**Comment by asudoh** [comment]
Need to port IBM/carbon-components-react#518 to here.

**Comment by kavyanekkalapu** [comment]
@asudoh Is the fix already merged in react repo? we use angular, is there anyway i can get fix quickly? we have a deadline this weekend, thanks.

**Comment by asudoh** [comment]
Hi @kavyanekkalapu you can try setting [`aria-owns`](https://developers.google.com/web/fundamentals/accessibility/semantics-aria/aria-labels-and-relationships#aria-owns) (and other ARIA attributes) that https://github.com/IBM/carbon-components-react/pull/518 sets. Thanks!

**Comment by kavyanekkalapu** [comment]
I asked this issue in avt channel and found out that this issue is Invalid, INS+DOWN need not be  reading hidden content such as tooltips. closing this issue.
